src/txmempool.cpp

```diff
@@ -55,10 +55,9 @@ double
 CTxMemPoolEntry::GetPriority(unsigned int currentHeight) const
```

Comment:
Everything that has "priority" in its name is scheduled to be removed, so this function as well.